mxcubeweb/core/adapter/wavelength_adapter.py

```diff
@@ -46,7 +49,8 @@ def set_value(self, value: HOActuatorValueChangeModel):
             self._ho.set_wavelength(float(value.value))
             return self.get_value()
         except Exception:
-            raise
+            msg = "Could not set value the wavelength: {ho.name}"
+            logging.getLogger("MX3.HWR").exception(msg)
```

Comment:
I recommend setting a global variable:

```python
log = logging.getLogger("MX3.HWR")
```

And then use like this

```suggestion
            log.exception(msg)
```

This would avoid repetitions. But I also recommend to do it even when there is no repetition. This is a very common practice. Only MXCuBE, that I know of, does things like they are done here now.

Reply:
For another future PR, what do you guys think of adding to the base class or application instead, so that one can do either `self.app.mxhwr.log` or `self.mxhwr.log` ?.

Reply:
> For another future PR, what do you guys think of adding to the base class or application instead, so that one can do either `self.app.mxhwr.log` or `self.mxhwr.log` ?.

My first impression is that it seems like it would be fine.

Reply:
Its perhaps more convenient compared to adding `log = logging.getLogger("MX3.HWR")` in each file

Reply:
Yes, seems like it would be more convenient.

_[But for me I usually put those logger lines at the top of each of my modules anyway (boilerplate code), so I am used to see them there.]_